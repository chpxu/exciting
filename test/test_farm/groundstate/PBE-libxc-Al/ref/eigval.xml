<?xml version="1.0" encoding="UTF-8"?>
<eigval>
  <kpt ik="1" vkl="1.250000000000e-1 1.250000000000e-1 1.250000000000e-1">
    <state ist="1" eigenvalue="-2.094095006618e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-2.093952420137e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-2.093952420137e0" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="-1.192867828182e-1" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="6.029923140365e-1" occupancy="0.000000000000e0"/>
    <state ist="6" eigenvalue="7.312934769980e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="7.312934769980e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="8.496896449951e-1" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="8.888723336114e-1" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="8.888723336114e-1" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="2" vkl="3.750000000000e-1 1.250000000000e-1 1.250000000000e-1">
    <state ist="1" eigenvalue="-2.094609891268e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-2.094155005915e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-2.094024655930e0" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="-3.392768067231e-2" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="3.723019869317e-1" occupancy="0.000000000000e0"/>
    <state ist="6" eigenvalue="5.402641234003e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="7.370901571528e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="8.087836896778e-1" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="8.104267443566e-1" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="9.106087798112e-1" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="3" vkl="6.250000000000e-1 1.250000000000e-1 1.250000000000e-1">
    <state ist="1" eigenvalue="-2.094896336930e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-2.094213888360e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-2.094140313335e0" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="9.102514460837e-2" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="1.742238657476e-1" occupancy="2.000000000000e0"/>
    <state ist="6" eigenvalue="5.765297611029e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="6.265043471018e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="6.942536660270e-1" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="8.485377659489e-1" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="9.590370559306e-1" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="4" vkl="8.750000000000e-1 1.250000000000e-1 1.250000000000e-1">
    <state ist="1" eigenvalue="-2.094370850304e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-2.094054853546e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-2.094025474111e0" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="-7.641021267208e-2" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="4.885085894328e-1" occupancy="0.000000000000e0"/>
    <state ist="6" eigenvalue="6.253828473499e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="6.799859398860e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="7.308637337058e-1" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="8.297333466808e-1" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="9.764842033463e-1" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="5" vkl="3.750000000000e-1 3.750000000000e-1 1.250000000000e-1">
    <state ist="1" eigenvalue="-2.094700172452e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-2.094249241094e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-2.094213345001e0" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="8.122022247896e-3" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="4.092690350490e-1" occupancy="0.000000000000e0"/>
    <state ist="6" eigenvalue="4.997801316466e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="5.527509535507e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="6.068687838754e-1" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="7.651863750824e-1" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="1.059315980090e0" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="6" vkl="6.250000000000e-1 3.750000000000e-1 1.250000000000e-1">
    <state ist="1" eigenvalue="-2.094721140655e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-2.094500744189e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-2.094292478957e0" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="1.708648416891e-1" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="2.519604603195e-1" occupancy="2.000000000000e0"/>
    <state ist="6" eigenvalue="3.435147956191e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="4.289831547083e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="7.382582007665e-1" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="8.993065748163e-1" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="9.684940380368e-1" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="7" vkl="8.750000000000e-1 3.750000000000e-1 1.250000000000e-1">
    <state ist="1" eigenvalue="-2.094756343670e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-2.094321322145e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-2.094143937301e0" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="4.984309475485e-2" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="2.925664620271e-1" occupancy="9.999951819887e-1"/>
    <state ist="6" eigenvalue="4.622621300361e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="5.425239763892e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="7.739434021108e-1" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="8.543428612114e-1" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="9.368068640861e-1" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="8" vkl="6.250000000000e-1 6.250000000000e-1 1.250000000000e-1">
    <state ist="1" eigenvalue="-2.094832285668e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-2.094381131356e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-2.094285573906e0" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="1.306466753230e-1" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="3.017571705654e-1" occupancy="0.000000000000e0"/>
    <state ist="6" eigenvalue="3.718900423389e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="5.164363192620e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="5.658305893358e-1" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="7.295601006127e-1" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="1.165176106451e0" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="9" vkl="3.750000000000e-1 3.750000000000e-1 3.750000000000e-1">
    <state ist="1" eigenvalue="-2.094833778619e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-2.094076289092e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-2.094076289092e0" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="8.179992974395e-3" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="2.549401650456e-1" occupancy="2.000000000000e0"/>
    <state ist="6" eigenvalue="6.999330806623e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="6.999330806623e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="7.743127542679e-1" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="7.743127542679e-1" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="8.362521727432e-1" occupancy="0.000000000000e0"/>
  </kpt>
  <kpt ik="10" vkl="6.250000000000e-1 3.750000000000e-1 3.750000000000e-1">
    <state ist="1" eigenvalue="-2.094768609209e0" occupancy="2.000000000000e0"/>
    <state ist="2" eigenvalue="-2.094472950153e0" occupancy="2.000000000000e0"/>
    <state ist="3" eigenvalue="-2.094177599814e0" occupancy="2.000000000000e0"/>
    <state ist="4" eigenvalue="1.318744862554e-1" occupancy="2.000000000000e0"/>
    <state ist="5" eigenvalue="2.127241892994e-1" occupancy="2.000000000000e0"/>
    <state ist="6" eigenvalue="3.851513699621e-1" occupancy="0.000000000000e0"/>
    <state ist="7" eigenvalue="5.898856880386e-1" occupancy="0.000000000000e0"/>
    <state ist="8" eigenvalue="6.588209621996e-1" occupancy="0.000000000000e0"/>
    <state ist="9" eigenvalue="9.593849736425e-1" occupancy="0.000000000000e0"/>
    <state ist="10" eigenvalue="1.035358142761e0" occupancy="0.000000000000e0"/>
  </kpt>
</eigval>
