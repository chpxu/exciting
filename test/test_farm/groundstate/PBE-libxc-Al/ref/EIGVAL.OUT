    10 : nkpt
    10 : nstsv
 
     1  0.1250000000      0.1250000000      0.1250000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -2.094095007       2.000000000    
     2  -2.093952420       2.000000000    
     3  -2.093952420       2.000000000    
     4 -0.1192867828       2.000000000    
     5  0.6029923140       0.000000000    
     6  0.7312934770       0.000000000    
     7  0.7312934770       0.000000000    
     8  0.8496896450       0.000000000    
     9  0.8888723336       0.000000000    
    10  0.8888723336       0.000000000    
 
 
     2  0.3750000000      0.1250000000      0.1250000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -2.094609891       2.000000000    
     2  -2.094155006       2.000000000    
     3  -2.094024656       2.000000000    
     4 -0.3392768067E-01   2.000000000    
     5  0.3723019869       0.000000000    
     6  0.5402641234       0.000000000    
     7  0.7370901572       0.000000000    
     8  0.8087836897       0.000000000    
     9  0.8104267444       0.000000000    
    10  0.9106087798       0.000000000    
 
 
     3  0.6250000000      0.1250000000      0.1250000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -2.094896337       2.000000000    
     2  -2.094213888       2.000000000    
     3  -2.094140313       2.000000000    
     4  0.9102514461E-01   2.000000000    
     5  0.1742238657       2.000000000    
     6  0.5765297611       0.000000000    
     7  0.6265043471       0.000000000    
     8  0.6942536660       0.000000000    
     9  0.8485377659       0.000000000    
    10  0.9590370559       0.000000000    
 
 
     4  0.8750000000      0.1250000000      0.1250000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -2.094370850       2.000000000    
     2  -2.094054854       2.000000000    
     3  -2.094025474       2.000000000    
     4 -0.7641021267E-01   2.000000000    
     5  0.4885085894       0.000000000    
     6  0.6253828473       0.000000000    
     7  0.6799859399       0.000000000    
     8  0.7308637337       0.000000000    
     9  0.8297333467       0.000000000    
    10  0.9764842033       0.000000000    
 
 
     5  0.3750000000      0.3750000000      0.1250000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -2.094700172       2.000000000    
     2  -2.094249241       2.000000000    
     3  -2.094213345       2.000000000    
     4  0.8122022248E-02   2.000000000    
     5  0.4092690350       0.000000000    
     6  0.4997801316       0.000000000    
     7  0.5527509536       0.000000000    
     8  0.6068687839       0.000000000    
     9  0.7651863751       0.000000000    
    10   1.059315980       0.000000000    
 
 
     6  0.6250000000      0.3750000000      0.1250000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -2.094721141       2.000000000    
     2  -2.094500744       2.000000000    
     3  -2.094292479       2.000000000    
     4  0.1708648417       2.000000000    
     5  0.2519604603       2.000000000    
     6  0.3435147956       0.000000000    
     7  0.4289831547       0.000000000    
     8  0.7382582008       0.000000000    
     9  0.8993065748       0.000000000    
    10  0.9684940380       0.000000000    
 
 
     7  0.8750000000      0.3750000000      0.1250000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -2.094756344       2.000000000    
     2  -2.094321322       2.000000000    
     3  -2.094143937       2.000000000    
     4  0.4984309475E-01   2.000000000    
     5  0.2925664620      0.9999951820    
     6  0.4622621300       0.000000000    
     7  0.5425239764       0.000000000    
     8  0.7739434021       0.000000000    
     9  0.8543428612       0.000000000    
    10  0.9368068641       0.000000000    
 
 
     8  0.6250000000      0.6250000000      0.1250000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -2.094832286       2.000000000    
     2  -2.094381131       2.000000000    
     3  -2.094285574       2.000000000    
     4  0.1306466753       2.000000000    
     5  0.3017571706       0.000000000    
     6  0.3718900423       0.000000000    
     7  0.5164363193       0.000000000    
     8  0.5658305893       0.000000000    
     9  0.7295601006       0.000000000    
    10   1.165176106       0.000000000    
 
 
     9  0.3750000000      0.3750000000      0.3750000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -2.094833779       2.000000000    
     2  -2.094076289       2.000000000    
     3  -2.094076289       2.000000000    
     4  0.8179992974E-02   2.000000000    
     5  0.2549401650       2.000000000    
     6  0.6999330807       0.000000000    
     7  0.6999330807       0.000000000    
     8  0.7743127543       0.000000000    
     9  0.7743127543       0.000000000    
    10  0.8362521727       0.000000000    
 
 
    10  0.6250000000      0.3750000000      0.3750000000     : k-point, vkl
 (state, eigenvalue and occupancy below)
     1  -2.094768609       2.000000000    
     2  -2.094472950       2.000000000    
     3  -2.094177600       2.000000000    
     4  0.1318744863       2.000000000    
     5  0.2127241893       2.000000000    
     6  0.3851513700       0.000000000    
     7  0.5898856880       0.000000000    
     8  0.6588209622       0.000000000    
     9  0.9593849736       0.000000000    
    10   1.035358143       0.000000000    
 
